init/systemd/agent_start_script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+dhclient eth0
+AGENT_GRPC_HOST=$(ip -4 addr show eth0 | grep inet | awk '{print $2}' | cut -d/ -f1)
```

Comment:
You kept name eth0?

Reply:
I have changed the name so it uses systemd. It is now `enp0s2`.